hardhat.config.ts

```diff
@@ -24,13 +36,18 @@ module.exports = {
   namedAccounts: {
     deployer: {
       default: 0,
+      alfajores: 0,
```

Comment:
Do we have indexed accounts available on these networks? Not seeing immediately where these are being derived from.

Reply:
Yes index 0 which is only what we need do exists. But i have now made it more explicit by providing actual addresses